targets/TARGET_STM/stm_spi_api.c

```diff
@@ -602,6 +606,24 @@ static const uint32_t baudrate_prescaler_table[] =  {SPI_BAUDRATEPRESCALER_2,
                                                      SPI_BAUDRATEPRESCALER_256
                                                     };
 
+static uint8_t spi_get_baudrate_prescaler_rank(uint32_t value) {
```

Comment:
Can you aling `{` to a new line - style consistency

Reply:
Fixed